integration/spark-common/src/main/scala/org/apache/carbondata/events/AlterTableEvents.scala

```diff
@@ -132,39 +136,33 @@ case class AlterTableRenameAbortEvent(carbonTable: CarbonTable,
 
 /**
  *
+ * @param sparkSession
  * @param carbonTable
- * @param carbonLoadModel
- * @param mergedLoadName
- * @param sQLContext
  */
-case class AlterTableCompactionPreEvent(sparkSession: SparkSession, carbonTable: CarbonTable,
-    carbonLoadModel: CarbonLoadModel,
-    mergedLoadName: String,
-    sQLContext: SQLContext) extends Event with AlterTableCompactionEventInfo
-
+case class AlterTableCompactionPreEvent(sparkSession: SparkSession, carbonTable: CarbonTable)
```

Comment:
Add description of each event, describe when it will be triggered

Reply:
ok